todoman/cli.py

```diff
@@ -224,49 +247,48 @@ def edit(ctx, id, todo_properties, interactive):
             setattr(todo, key, value)
 
     if interactive or (not changes and interactive is None):
-        ui = TodoEditor(todo, ctx.obj['db'].lists(), ctx.obj['formatter'])
+        ui = TodoEditor(todo, ctx.db.lists(), ctx.ui_formatter)
```

Comment:
Does this make sense? Do you think people will do that? Do we *care* about that scenario?